groupAnagrams.js

```diff
@@ -0,0 +1,44 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/group-anagrams/
+
+const groupAnagrams = (strs) => {
+  let strsArray = strs;
+  const result = [];
+  while (strsArray.length > 0) {
+    const grouppedAnagrams = [strsArray[0]]; // "eat"
+    for (let j = 1; j < strsArray.length; j += 1) {
+      if (strsArray[0].length !== strsArray[j].length) {
+        continue;
+      }
+      let letters = strsArray[j].split(""); // t e a
+      let strsArrItem = strsArray[0].split(""); // e a t
+      while (letters.length > 0) {
+        if (
+          strsArrItem.includes(letters[0]) &&
+          letters.length === strsArrItem.length
+        ) {
+          const savedLetter = letters[0];
+          letters = letters.filter((letter) => letter !== savedLetter[0]); // [e, a]
+          strsArrItem = strsArrItem.filter((item) => item !== savedLetter[0]); // [e, a]
```

Comment:
If `savedLetter` is a letter (a character), what would be the `savedLetter[0]` ? Is this a typo or am I missing something ?

Reply:
It is a mistake. Corrected with [cbff8a1](https://github.com/Femalopper/tasks/pull/6/commits/cbff8a100ddec7aa1703badb7f3daf9dea370d20)